Documentation/concepts/matching.md

```diff
@@ -7,6 +7,11 @@ Matchers by default are also responsible for keeping the database of vulnerabili
 The matcher API is designed to be called often and will always provide the most up-to-date VulnerabilityReport when queried. This VulnerabilityReport summaries both a manifest's contents and any vulnerabilities affecting the contents.
 
 See our [api](../howto/api.md) guide to learn how to view our api specification and work with the Matcher api.
+# Remote Matching
+A remote matcher behaves similarly to a matcher except that it uses api calls to fetch vulnerabilty data to a provided IndexReport. Remote matchers are usefull when it is not possible to persist data from a given source into the database.
+
+The CRDA remote matcher is responsible for fetching vulnerabilities from Red Hat CRDA. CRDA has the best in industry vulnerability data powered by Snyk. This data cannot be persisted into the db hence the CRDA matcher makes api calls based on the IndexReport to fetch vulnerability data.
+This matcher by default serves 100 requests per minute. Each request can have up to 10 packages for optimum performance. The CRDA matcher comes with a configurable option and we can accommodate a much higher rate of requests. Please fill out [CRDA-Request-Form](https://docs.google.com/forms/d/e/1FAIpQLSfJ1aoOfi0tQ6_o0srEbYL-cSwzo_MyDJQnwcFG9IYVlNXJrA/viewform?usp=sf_link)to avail the configuration details. This can then be used in the clair's config file.
```

Comment:
Thanks for the updates, I'd just say:
"used in ~the~ ~c~Clair's config file"
and a space after the form link.

Everything else LGTM

Reply:
@crozzy Made the requested changes. Thanks for the review comments, if there is nothing else may be you could add approve this pr.